docs/troubleshooting/index.md

```diff
@@ -40,8 +40,14 @@ This endpoint can be used on both the db nodes as well as the coordinator/query
 
 To use this, simply run the following on either the M3DB debug listen port or the regular port on M3Coordinator.
 
+You may need to include the following headers:
+
+--8<--
```

Comment:
What's this mean? Renders weirdly when I click the markdown view.

Reply:
yea it's as mkdocs thing, that will embed other pages. we do something similar for https://m3db.github.io/m3/operational_guide/placement_configuration/

see https://raw.githubusercontent.com/m3db/m3/master/docs/operational_guide/placement_configuration.md